src/components/CompleteOptionsToggles.tsx

```diff
@@ -120,10 +125,10 @@ export function CompleteOptionsToggles({
               {purposeToDescription.Essential?.description}
             </p>
           </span>
-          {orderedSelections.map(([purpose, isChecked]) => (
+          {orderedSelections.map(([purpose, isChecked], idx) => (
```

Comment:
Same here wrt wrapping in a fieldset or group role

Reply:
ditto my above comment